component-maps.yml

```diff
@@ -565,6 +566,13 @@ docker_image:
     release_component: true
     independent_component: false
 
+  mender-gateway:
+    git:
+    - mender-gateway
+    docker_container:
+    - mender-gateway
+    release_component: true
+
```

Comment:
there is one on [this](https://github.com/mendersoftware/integration/pull/2603/files#diff-37a5a19da4dabae31f06c276f64ff61b5d12bc2ed734bd377000e99d69142d38R787) line (below) does it matter?

Reply:
The are there are three top level objects in this file: Git repos, container image names, and container names. The one you are linking is the already existing Git repo and the one we are adding here is the container image.